src/applications/personalization/profile-2/components/personal-information/PersonalInformation.jsx

```diff
@@ -58,8 +62,13 @@ const PersonalInformation = ({
       >
         Personal and contact information
       </h2>
-      {showNotAllDataAvailableError && <MyAlert />}
-      <PersonalInformationContent />
+      <DowntimeNotification
+        render={handleDowntimeForSection('personal and contact')}
+        dependencies={[externalServices.mvi, externalServices.vet360]}
```

Comment:
Makes sense to me - we're watching for `mvi` in Personal Information and 'vet360` in Contact Information for profile 1 👍